### ISSUE TITLE
Nightly release required?

### ISSUE THREAD

**Issue by deeso** [naia-rs/naia#27]
Hello, I was trying out the demo code, and I got the following compile error:
```
cargo run --features "use-webrtc"
   Compiling naia-server-socket v0.7.0
   Compiling naia-basic-demo-shared v0.1.0 (/home/apridgen/Documents/rust_game_dev/naia/demos/basic/shared)
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> demos/basic/shared/src/lib.rs:1:12
  |
1 | #![feature(const_type_id)]
  |            ^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0554`.
error: could not compile `naia-basic-demo-shared` due to previous error
warning: build failed, waiting for other jobs to finish...
error: build failed

```

I am not a good rust software developer, so here goes a dumb question.  Do I need to be using nightly (unstable) release to use this framework?

Thanks for putting this project together nonetheless.

**Comment by ypoonse** [comment]
Hello,

As a relative new rust developer, I know that you can install the nightly toolchain, and then add "+nightly", i.e., 

```bash
cargo +nightly run --features "use-webrtc"
```

The reason I'm visiting this issue is for a related question: what would be needed so this project can use stable rust? Are there plans to support that? (or does the author know when const_type_id would be stabilized)?

**Comment by ypoonse** [comment]
That being said, I haven't succeeded in running the demo as explained in the README, ``npm start`` also compiles rust code (throught wasm-pack?), and I haven't figured out how to have that command compile code using nightly.

**Comment by naia-rs** [comment]
Should be fixed in 0.7.3
